.github/workflows/well-known-environment.yml

```diff
@@ -149,6 +149,8 @@ jobs:
           dev-account-role: ${{ secrets.UHD_TERRAFORM_ROLE_DEV }}
           test-account-role: ${{ secrets.UHD_TERRAFORM_ROLE_TEST }}
           uat-account-role: ${{ secrets.UHD_TERRAFORM_ROLE_UAT }}
+          auth-dev-account-role: ${{ secrets.UHD_TERRAFORM_ROLE_AUTH_DEV }}
```

Comment:
I suspect that this role won't exist yet, so we've got the chicken & egg situation I was talking about before. 
I think we'll likely need to deploy to those envs from our machines the first time around, and then we can pass in the role ARN to these secrets